src/web_app/html/params.html

```diff
@@ -89,8 +89,5 @@
     <a href="//github.com/GoogleChrome/webrtc" title="View the repository" id="viewSource">github.com/GoogleChrome/webrtc</a>
 
   </div>
-
```

Comment:
I have no clue if it is needed or not, and did not spend any brain on it.. i just happen to be browsing changes (when apprtc was not working) and though it was strange a fix for .css path touched this line.